src/core/load-fragment.c

```diff
@@ -3939,7 +3941,14 @@ int config_parse_bind_paths(
                         return 0;
                 }
 
-                s = source;
+                r = unit_full_printf(u, source, &sresolved);
+                if (r < 0) {
+                        log_syntax(unit, LOG_ERR, filename, line, r,
+                                   "Failed to resolved specifiers in \"%s\", ignoring: %m", source);
+                        return 0;
```

Comment:
I wonder why we return `0` here if `unit_full_printf` returns `-ENOMEM`. I think we should call `return log_oom()`.